test/jdk/javax/swing/JEditorPane/4694598/bug4694598.java

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright (c) 2002, 2023, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+/* @test
+ * @bug 4694598
+ * @key headful
+ * @summary JEditor pane throws NullPointerException on mouse movement.
+ * @library ../../regtesthelpers
+ * @build JRobot
```

Comment:
The test doesn't seem to use any of the extended functionality provided by JRobot, so the standard `java.awt.Robot` can be used instead.

Reply:
I do not see any harm in using JRobot, it is a Swing test after all.

Reply:
No harm but a dependency on a library which isn't used.

Reply:
I am very much against including libraries that aren't needed.
If this test isn't using any JRobot functionality, please remove it.

Reply:
Ok, removed.